package-lock.json

```
@@ -9878,11 +9849,22 @@
         "vscode-uri": "^3.0.2"
       }
     },
+    "langium-arithmetics-dsl": {
+      "version": "file:examples/arithmetics",
+      "requires": {
+        "chalk": "4.1.2",
```

Minor: For some reason your version of npm doesn't pick it up on its own, but this should be:
```suggestion
        "chalk": "^4.1.2",
```

---

manually edited the lock file - strange it didn't pick up